analyzers/tests/SonarAnalyzer.Test/Rules/AspNet/AvoidUnderPostingTest.cs

```diff
@@ -53,6 +53,13 @@ public class AvoidUnderPostingTest
             .WithOptions(ParseOptionsHelper.FromCSharp9)
             .Verify();
 
+    [TestMethod]
+    public void AvoidUnderPosting_CSharp11() =>
+    builder.AddPaths("AvoidUnderPosting.CSharp11.cs")
+        .AddReferences(NuGetMetadataReference.SystemTextJson("7.0.4"))
+        .WithOptions(ParseOptionsHelper.FromCSharp11)
```

Comment:
Why do we need C# 11 specifically here?

Reply:
Yeah - you are right.
JsonRequired was introduced with NET7 that's why I put it in its own file - but it's not a language feature. My bad.
I'll merge it with C# 12.

Reply:
aaaaaaaaaaaaaaaaaaaaaaaaaawesome